code/modules/modular_computers/computers/machinery/console_presets.dm

```diff
@@ -74,43 +91,41 @@
 /obj/machinery/modular_computer/preset/curator
 	name = "curator console"
 	desc = "A stationary computer. This one comes preloaded with art programs."
-	starting_programs = list(
+	preinstalled_programs = list(
 		/datum/computer_file/program/portrait_printer,
 	)
 
 // ===== CARGO CHAT CONSOLES =====
 /obj/machinery/modular_computer/preset/cargochat
 	name = "cargo interfacing console"
 	desc = "A stationary computer that comes pre-loaded with software to interface with the cargo department."
-	starting_programs = list(
+	startup_program = "dept_order"
```

Comment:
As a thought: These should be
`/datum/computer_file/program/department_order::filename`
, etc. 
So if their filename is updated these don't get left behind mistakenly

Reply:
Yeah, I was torn about specifying a datum or filename based on how the proc loads the file, this suggestion is better.